### PR TITLE
Simple collisions

### PR DESCRIPTION
**Pull request by ChielBruin**
Added simple collision detection between entities and the player.

### CODE REVIEW

**Comment by Diocruel** [comment]
Looks great! :)

File: src/test/java/nl/tudelft/contextproject/model/EntityTest.java [hunk 1]
```diff
@@ -47,4 +55,42 @@ public void testSetState() {
 		entity.setState(EntityState.ALIVE);
 		assertEquals(entity.getState(), EntityState.ALIVE);
 	}
+	
+	/**
+	 * Test if collision detection returns true when it should.
+	 */
+	@Test
+	public void testPlayerCollisionTrue() {
+		TestUtil.mockGame();
```
Comment (Taeir) [inline src/test/java/nl/tudelft/contextproject/model/EntityTest.java:64]: Should preferrably be done in an "@BeforeClass" method.
Take a look at WebTestBase for an example of a safe set/unset method.

**Comment by Taeir** [comment]
I placed a comment about moving `TestUtil.mockGame()` to an `@BeforeClass` method.

Apart from that, this looks clean and simple, and can be merged in my opinion.

**Comment by ChielBruin** [comment]
I tried moving it to a beforeClass method, but this caused tests to fail..

**Comment by Geweldig** [comment]
In that case it's mergeable, at least in my opinion.

**Comment by Diocruel** [comment]
All right, lets merge it.

**Pull request merged**